tracee-rules/main.go

```diff
@@ -93,6 +93,14 @@ func main() {
 				Name:  "webhook",
 				Usage: "HTTP endpoint to call for every match",
 			},
+			&cli.StringFlag{
+				Name:  "webhook-template",
+				Usage: "path to a gotemplate for formatting webhook output",
+			},
+			&cli.StringFlag{
+				Name:  "content-type",
```

Comment:
can you please change to webhook-content-type?

Reply:
Sure: https://github.com/aquasecurity/tracee/pull/582/commits/35fa90dc65a7fc5b0a8b9e492456f61369b795fc